### ISSUE TITLE
Documentation?

### ISSUE THREAD

**Issue by v1rtl** [clayrisser/reactant#7]
In order to use this package I need to understand **what it does** and **how to use it**. Please setup docs using github.io or any other way to make docs and host them.

**Comment by clayrisser** [comment]
Absolutely! I'm in the process of doing it. My documentation process will be using sphinx to generate docs that get fed through the sphinx markdown plugin that get fed into gatsby. You can see these projects below.

https://github.com/codejamninja/sphinx-markdown-builder
https://github.com/codejamninja/node-sphinxdoc
https://github.com/codejamninja/gatsbydoc

`TypeScript -> typedoc -> sphinx-js -> sphinx -> sphinx-markdown-builder -> markdown -> gatsby -> SUPER AWESOME DOCS`

**Comment by v1rtl** [comment]
Thanks for information, I'll be waiting for documentation
